lib/rack/tracker/bing/template/bing.erb

```diff
@@ -0,0 +1,23 @@
+<% if events.any? %>
+<script src="//bat.bing.com/bat.js" async=""></script>
+<script>
+  <% events.each do |event| %>
+  function reportCustomEvent() {
+    window.uetq = window.uetq || []; window.uetq.push({ 'ec': '<%= event.category %>', 'ea': '<%= event.action %>', 'el': '<%= event.label %>', 'ev': <%= event.value %> });
+  }
+  <% end %>
+</script>
+<% end %>
+
+
+<script>(function (w, d, t, r, u) {
+  var f, n, i;
+  w[u] = w[u] || [], f = function () {
+    var o = {ti: "<%= tracker %>"};
+    o.q = w[u], w[u] = new UET(o), w[u].push("pageLoad")
+  }, n = d.createElement(t), n.src = r, n.async = 1, n.onload = n.onreadystatechange = function () {
+    var s = this.readyState;
+    s && s !== "loaded" && s !== "complete" || (f(), n.onload = n.onreadystatechange = null)
+  }, i = d.getElementsByTagName(t)[0], i.parentNode.insertBefore(n, i)
+})(window, document, "script", "//bat.bing.com/bat.js", "uetq");
```

Comment:
I don't know bing, but was wondering if this is loading the same `bat.js` here as in line 2?

Reply:
Again I'm just going by what the "[docs](http://bingadsuet.azurewebsites.net/UETDirectOnSite_ReportCustomEvents.html)" say and they have bat.js loaded as part of the tracker code and then again when defining events...

I might test it using my branch and report back! 😄

Reply:
yep, either way I guess that's not a big problem. we only should look into the events - if they are fired.